source/_integrations/webdav.markdown

```diff
@@ -0,0 +1,51 @@
+---
+title: WebDAV
+description: Instructions on how to setup a WebDAV location to be used with backups.
+ha_release: 2025.2
+ha_category:
+  - Backup
+ha_iot_class: Cloud Polling
+ha_config_flow: true
+ha_domain: webdav
+ha_codeowners:
+  - '@jpbede'
+ha_integration_type: service
+related:
+  - docs: /common-tasks/general/#backups
+    title: Backups
+ha_quality_scale: bronze
+---
+
+This integration allows you to use a [WebDAV](https://en.wikipedia.org/wiki/WebDAV) compatible location for [Home Assistant Backups](/common-tasks/general/#backups).
+
+## Installation
+
+{% include integrations/config_flow.md %}
+{% configuration_basic %}
+URL:
+  description: "URL of the WebDAV server. Common examples are provided below."
+Username:
+  description: "Username for the WebDAV server."
+Password:
+    description: "Password for the WebDAV server."
+Backup path:
+  description: "Path to the folder where the backups should be stored. The path is relative to the root of the WebDAV server."
+Verify SSL:
+    description: "Verify the SSL certificate of the WebDAV server."
+    default: true
```

Comment:
```suggestion
```

the `default` key is not evaluated anymore

![image](https://github.com/user-attachments/assets/266fbec1-3388-4db8-87bb-83c7d65f9655)